src/azul/uuids.py

```diff
@@ -57,28 +61,36 @@ def validate_uuid_prefix(uuid_prefix: str) -> None:
     # The empty string is a valid prefix
     >>> validate_uuid_prefix('')
 
-    >>> validate_uuid_prefix('8f53')
+    >>> validate_uuid_prefix('8')
 
-    # A complete UUID is a valid prefix
-    >>> validate_uuid_prefix('8f53d355-b2fa-4bab-a2f2-6852d852d2ec')
+    >>> validate_uuid_prefix('8f538f53')
 
-    >>> validate_uuid_prefix('8F53')
+    >>> validate_uuid_prefix('8f538f5-')
     Traceback (most recent call last):
     ...
-    azul.uuids.InvalidUUIDPrefixError: '8F53' is not a valid UUID prefix.
+    azul.RequirementError: Remove the trailing `-` from the uuid prefix: 8f538f5-
+
+    >>> validate_uuid_prefix('8f538f-')
+    Traceback (most recent call last):
+    ...
+    azul.RequirementError: Remove the trailing `-` from the uuid prefix: 8f538f-
+
+    >>> validate_uuid_prefix('8f538f53a')
+    Traceback (most recent call last):
+    ...
+    azul.uuids.InvalidUUIDPrefixError: '8f538f53a' is not a valid UUID prefix.
 
-    >>> uuid = '8f53d355-b2fa-4bab-a2f2-6852d852d2ec'
-    >>> # noinspection PyUnresolvedReferences
-    >>> prefixes = [uuid[:n] for n in range(len(uuid))]
-    >>> for prefix in prefixes: validate_uuid_prefix(prefix)
+    # A complete UUID is a valid prefix
+    >>> validate_uuid_prefix('8f53d355-b2fa-4bab-a2f2-6852d852d2ec')
 
-    >>> validate_uuid_prefix('aaaaaaaa-aaaa-')
-    >>> validate_uuid_prefix('aaaaaaaa-aaaa-a')
+    >>> validate_uuid_prefix('8F53')
```

Comment:
Can you move this back so as to minimize the diff?

Reply:
self-note: inspect diff when other fixups land.